ocaml/idl/datamodel.ml

```diff
@@ -505,6 +505,9 @@ let _ =
   error Api_errors.cannot_contact_host ["host"]
     ~doc:"Cannot forward messages because the host cannot be contacted.  The host may be switched off or there may be network connectivity problems." ();
 
+  error Api_errors.network_is_unreachable ["address"]
+    ~doc:"Cannot contact the Host on the specified network" ();
```

Comment:
The network might be reachable: the failure to make a TLS connection might be for some other reason.
I feel it's worth mentioning the port as well as the IP address.